src/libs/storage.ts

```diff
@@ -8,6 +8,7 @@ type StorageKey =
   | 'has_accepted_cookie'
   | 'has_seen_eligible_card'
   | 'phone_validation_code_asked_at'
+  | 'PASSCULTURE_REFRESH_TOKEN'
```

Comment:
case ?

Reply:
Not applicable anymore puisque j'ai supprimé le commit associé